sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -1,30 +1,39 @@
 package com.smartdevicelink.protocol;
 
+
+import android.util.Log;

```

Comment:
I missed that. We should pass those params back. I don't think they are helpful to long unless a developer is debugging. Maybe for now just leave a //TODO and with the modifications for the video streaming params we can pass back the rejected params.